external-crates/move/crates/move-compiler/src/typing/translate.rs

```diff
@@ -3769,12 +3765,12 @@ fn method_call(
                 edotted.for_autocomplete = true;
                 let err_ty = context.error_type(loc);
                 let dot_output =
-                    resolve_exp_dotted(context, DottedUsage::Borrow(false), loc, edotted);
+                    resolve_exp_dotted(context, DottedUsage::Borrow(false), edotted.loc, edotted);
```

Comment:
Is this related or unrelated fix?

Reply:
All of the span stuff is quite related. I'll provide a write-up in the mian PR when I publish it for review.